packages/relay/src/lib/eth.ts

```diff
@@ -792,7 +792,14 @@ export class EthImpl implements Eth {
           gas = call.gas;
         }
       }
-      
+
+      // Gas limit for `eth_call` is 50_000_000, but the Hedera network limit is 15_000_000
+      // With values over 15_000_000 the call will fail with BUSY error
+      if (gas > 15_000_000) {
+        this.logger.trace(`${requestIdPrefix} eth_call gas amount (${gas}) exceeds network limit, capping gas to 15_000_000`);
+        gas = 15_000_000;
```

Comment:
nit: could have made `15_000_000` a const
Can make this change in a future PR, no biggie

Reply:
I'll still probably have to merge main at some point to fix the dapp tests, I can push that change too